clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/utils/PublicKeyUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.abcxyz.lumberjack.auditlogclient.utils;
```

Comment:
Test util shouldn't be in non-test code? Load the file twice is fine.

Reply:
Fixed.